sopel/config/types.py

```diff
@@ -216,31 +219,43 @@ def __get__(self, instance, owner=None):
             # (and is really just more intuitive) to return the descriptor
```

Comment:
Can't touch the line above this (GitHub staff, please read and implement isaacs/github#284)…

I think "session class" is a typo. Could be wrong, but it _sounds_ incorrect. Doesn't this just mean "from a section class, not a section instance"? It's confusingly worded.

Reply:
Yup it's a typo, I'll fix that.

Reply:
Fixed that for you.